### PR TITLE
Sophie Cytoscape Assignment Submission

### PR DESCRIPTION
**Pull request by sophiegoldman**
Dear Professor Reynolds,

Here is my submission for the 4th digital tools assignment!

Best,
Sophie

### CODE REVIEW

File: _posts/2021-04-19-Networked Dystopia.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+Through this process, I began to realize the surprising complexity of my question. Some movies, especially those that took place in space alone, were relatively straightforward to classify. Others, however, became much harder, as I found that two movies set in the same physical location might have entirely different *settings* when considering the impact of the movie's plot and time period. This led me to create multiple classification categories that separated movies between those describing an alternate past, the present, and the future. I chose to distinguish the future separately from the present as a distant future often exhibits different visual and societal features than the audience's more familiar modern setting, even if the name of the setting has not changed. I also chose to separate the setting classifications into global, national, and city-specific levels in order to see if particular locations were more commonly used as dystopian settings. I only included edges at the most specific levels, such that I only included an edge between the film and its city, and not the city's country, in cases where a specific city was identifiable as a setting.
```
Comment (melissabreynolds) [inline _posts/2021-04-19-Networked Dystopia.md]: This is a great description of how you defined your data set and made decisions about how to link your analytical categories (settings) with your data set.

File: _posts/2021-04-19-Networked Dystopia.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+[The spreadsheet of movies and their settings can be found here.](https://docs.google.com/spreadsheets/d/1aJKW8D20lYVDX--59RklBFRlqNi0tgz_dzQ4i5_sINY/edit?usp=sharing)
+
+#### Below is the dictionary containing the labels for the settings and a description of what criteria were used to assign that setting to a given movie:
```
Comment (melissabreynolds) [inline _posts/2021-04-19-Networked Dystopia.md]: I really appreciate you adding this data dictionary--I can see that your work in Introduction to Digital Humanities is definitely informing this particular assignment. This is a really great model for other students or even readers of the blog (and believe it or not, we do have readers) to see the kinds of analytical thinking and organizational principles that go into devising a standard set of criteria with which to analyze a data set.

**Review by melissabreynolds — DISMISSED** [review]
As always, Sophie, this is a really wonderful analysis of the individual steps in developing your data set, devising your organizational principles, refining your analytical categories, and thinking through the conclusions of your network mapping. I love the choice of data set, too, because unlike the relationship between movies and actors that we looked at in class, as you say in your analysis, the relationship between dystopian films and their settings speaks so clearly to the sort of impact these films seek to make with their audiences. Your choice to subdivide your network into these smaller constellations of nodes and edges also really makes the range of settings clear to your reader. Very nice work.

I am really excited to see what you and Hope come up with for the final project, and I am more than happy to talk further if you two have questions or concerns.

Can I go ahead and merge this post to publish it to the site?

Grade: A

**Comment by sophiegoldman** [comment]
Thank you for the feedback! I really enjoyed putting the project together - it would be great if you could merge it to the site!

Best,
Sophie

**Review by melissabreynolds — APPROVED** [review]

**Pull request merged**